src/libraries/System.Text.Json/tests/Common/ConstructorTests/ConstructorTests.ParameterMatching.cs

```diff
@@ -376,8 +376,8 @@ public async Task ClassWithPrimitives_Parameterless()
             point.SecondInt = 348943;
             point.FirstString = "934sdkjfskdfssf";
             point.SecondString = "sdad9434243242";
-            point.FirstDateTime = DateTime.Now;
-            point.SecondDateTime = DateTime.Now.AddHours(1).AddYears(1);
+            point.FirstDateTime = DateTime.UtcNow;
```

Comment:
Thanks for submitting the PR, but either `Now` or `UtcNow` are sources of nondetermism that should best be kept away from unit testing. Would it make sense to replace with a fixed value? It could be kept behind a propety called `DateTimeTestHelpers.FixedDateValue` or something along these lines.

Reply:
It's less of a concern here because the tests aren't using them for the actual value (which is covered elsewhere), but rather just to have something to serialize/deserialize.  That aside, all the references can be changed, yes - I was just making the simplest possible change initially.

> It could be kept behind a propety called `DateTimeTestHelpers.FixedDateValue` or something along these lines.

I think I'd look at adding it to `Common`, if there isn't one there already.  I really wish one of the `IClock` proposals would get in, so we could have a test clock.

Reply:
> That aside, all the references can be changed, yes - I was just making the simplest possible change initially.

Up to you if you want to go the extra mile, I'm fine with merging as-is since it fixes the issue at hand.

Reply:
I'll make a pass tonight.

Reply:
@eiriktsarpalis - Added a value to the `CommonTestPath`